state/upgrades_test.go

```diff
@@ -100,7 +100,7 @@ func (s *upgradesSuite) TestStripLocalUserDomainModels(c *gc.C) {
 	delete(initialModel, "txn-revno")
 	initialModel["owner"] = "test-admin"
 
-	// TODO (mattyw, cmars) We need to confirm that setting sla in this test
+	// TODO (mattyw, cmars) We need to confirm that setting sla and meter-status in this test
```

Comment:
@cmars no upgrade is required for SLAs or meter status. This test is actually checking completely seperate behaviour (local domains like "@local" being stripped from user names. However it includes checking against a bson representation of the model, which now includes sla and meter status